### ISSUE TITLE
[Command Area]: Voice Access users are not able to type most of the symbols and punctuation marks in the command area.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13821]
### Windows Terminal version

1.15.2003.0

### Windows build number

10.0.25179.1000

### Other Software

**Test Environment:**
OS: Window 11 22H2 OS Build 25179.1000
App: Windows Terminal Preview
AT: Voice Access

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Voice Access AT.
3. In Command Prompt command area, try to type Asterisk, Backslash, Forward slash, Underscore, Percent Sign, Pound/Number sign, Plus, less than sign, Greater than sign, Dollar sign, Comma, Colon, Semicolon Begin/Open Single quote, End/Close single quote, Left/Open parentheses, Right/Close Parentheses, Open bracket, Close bracket, Left/Open brace, 'Right/Close brace' symbols and observe the issue.

**User Experience:**
People with limited mobility who use Voice Access will have bad user experience as they will not be able to execute those commands where they need to use special symbols.

**Guideline Reference:**
EN 301 549 V3.2.1: 11.06.02 No disruption of accessibility features

**Attachments:**
[Special symbols.zip](https://github.com/microsoft/terminal/files/9400144/Special.symbols.zip)

Attachment Note: Not attached Code Snippet as not able to inspect Terminal Preview app with AI and Inspector tool.

### Expected Behavior

Voice Access users should be able to type all of the symbols and punctuation marks in the command area.

### Actual Behavior

Voice Access users are not able to type below mentioned symbols and punctuation marks in the command area.

- Asterisk, Backslash, Forward slash, Underscore, Percent Sign, Pound/Number sign, Plus, less than sign, Greater than sign, Dollar sign, Comma, Colon, Semicolon Begin/Open Single quote, End/Close single quote, Left/Open parentheses, Right/Close Parentheses, Open bracket, Close bracket, Left/Open brace, 'Right/Close brace'.

**Observation:**
Voice Access users can use Onscreen keyboard to perform same task.

To learn more about Voice Access and how an application can support its different modes, Refer https://www.osgwiki.com/wiki/VoiceAccess_Testing

Voice Access Commands page: https://sway.office.com/Q5JvoIGBI5glyoKw?ref=Link

**Comment by carlos-zamora** [comment]
@v-rpundir I'm getting these issues with other apps like Notepad and Microsoft Word too. Voice Access just shows an error saying "Please tell us which field you want to focus on.". Thus, this seems like it should be a bug filed on Voice Access itself instead of Windows Terminal or any underlying app.

Do you have guidance for any next steps here? Can we close this bug out? I don't think there's anything we can even do on our end.

**Comment by v-rpundir** [comment]
@carlos-zamora we will follow external bug process here, so I am forwarding this bug to Voice Access team and will update external tracking ID when I get it.

You can resolve or close this bug with resolved reason External.

**Comment by v-rpundir** [comment]
Voice Access Tracking ID: [Bug 41814107](https://microsoft.visualstudio.com/OS/_workitems/edit/41814107): [Voice Access] : Voice Access users are unable to input symbols and punctuation marks in the terminal
